src/inc/eetwain.h

```diff
@@ -161,7 +161,7 @@ enum
 };
 
 #ifndef DACCESS_COMPILE
-
+#ifdef _TARGET_X86_
```

Comment:
This method does not make sense for !WIN64EXCEPTIONS . I think it would more appropriate to put it under `#ifndef WIN64EXCEPTIONS`.